{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/vue/components/mobile-nav/MobileNav.css

```diff
@@ -39,7 +39,7 @@
   letter-spacing: calc(40em / 1000);
   text-transform: uppercase;
 
-  color: blue;
+  color: #f00;
```

Comment:
Yea, the colour doesn't really matter, styelint just didn't like named colours and i'm not 100% sure what `blue` is as a hex off the top of my head.